_posts/2017-5-31-release-1.3.0.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title:  "Apache Flink 1.3.0 Release Announcement"
+date:   2017-06-01 19:00:00
```

Comment:
Please use 12:00:00. Otherwise the date of the announcement is compiled on different days in different timezones.